libs/nx-react-workshop/src/migrations/complete-lab-2/complete-lab-2.ts

```diff
@@ -33,166 +27,10 @@ export default async function update(tree: Tree) {
     style: 'scss',
     unitTestRunner: 'jest',
     routing: true,
+    addPlugin: true,
```

Comment:
Ensures we use crystallized config which is what the user will see when they run the commands directly.